py-polodb/Cargo.toml

```diff
@@ -17,6 +18,8 @@ name = "rust_polodb"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-# polodb_core = "5.1.0"
-polodb_core = { path = "../src/polodb_core", default-features = false  }
+# polodb-librocksdb-sys = { path = "../src/librocksdb-sys", version = "9.0.0-alpha.1", features = ["default", "mt_static"] }
+
+polodb_core = "5.1.3"
```

Comment:
Why do we want to use the fixed version instead of "path"?

Reply:
To decouple their evolution and not suffering from braking change in one into another